### PR TITLE
Fixed remove-node exception and disabled leave command on single node…

### LINKED ISSUE #4006: Non-HA single node, leaving node removes all pods
#### Summary
Iam running the microk8s in ubuntu in no high availability i.e. there is a single node onto the same machine where it is installed. I updated the certificates and then i issue following command:
`sudo microk8s leave`

It give following messages:

```
Generating new cluster certificates.
Waiting for node to start. 
```

and then i stopped microk8s and then started again, the node appeared  however all of my pods / namespaces are gone, how to recover?

#### What Should Happen Instead?
All the pods should retain

#### Reproduction Steps
Already described above

#### Can you suggest a fix?

[https://wetransfer.com/downloads/ee891f9d62bd9ffd7fdac2f9597e638f20230529135310/bf5d38484b8a54a7107a5447153c884820230529135327/7b32a3](url)

### PR DESCRIPTION
**Pull request by berkayoz**
Fixed remove-node exception and disabled leave command on single node clusters

Closes #4006

### CODE REVIEW

**Review by sachinkumarsingh092 — APPROVED** [review]
Tested locally an it works fine. LGTM.

File: scripts/wrappers/remove_node.py [hunk 1]
```diff
@@ -25,7 +25,13 @@
 def remove_dqlite_node(node, force=False):
     try:
         # If node is an IP address, find the node name.
-        if type(ip_address(node)) is IPv4Address:
+        is_node_ip = False
+        try:
+            is_node_ip = True if ip_address(node) is IPv4Address else False
+        except ValueError:
+            pass
```
Comment (neoaggelos) [inline scripts/wrappers/remove_node.py:32]: Perhaps this is an easier way to write the same?

```suggestion
        is_node_ip = True
        try:
            ip_address(node)
        except ValueError:
            is_node_ip = False
```

**Review by neoaggelos — APPROVED** [review]
LGTM, just a minor comment

**Review by neoaggelos — CHANGES_REQUESTED** [review]
Sorry, wrong button

File: scripts/wrappers/remove_node.py (hunk 1 above)
Comment (berkayoz) [reply scripts/wrappers/remove_node.py]: Oh, I missed the `type()` check. Thanks for the review!

**Review by neoaggelos — APPROVED** [review]

**Pull request merged**